stripe/src/main/java/com/stripe/android/view/CardNumberEditText.kt

```diff
@@ -363,8 +363,14 @@ class CardNumberEditText internal constructor(
                 (isValid && accountRange != null)
             )
 
-        private fun isPastedPan(start: Int, cardNumber: CardNumber.Unvalidated): Boolean {
-            return start == 0 && cardNumber.normalized.length >= CardNumber.MIN_PAN_LENGTH
+        private fun isPastedPan(
+            start: Int,
+            before: Int,
+            count: Int,
+            cardNumber: CardNumber.Unvalidated
+        ): Boolean {
+            return count > before && start == 0 &&
```

Comment:
nit: took me a minute to figure out what these params all mean and why we want `count > before`. Maybe a comment would help here.

Reply:
ok, will do in follow up to unblock CI